schedule/yast/transactional_server/create_hdd_transactional_server@s390x.yaml

```diff
@@ -0,0 +1,43 @@
+---
+name: create_hdd_transactional_server
+description: >
+    Installation of a Transactional Server which uses a read-only
+    root filesystem to provide atomic, automatic updates of a
+    system without interfering with the running system.
+vars:
+    SCC_ADDONS: tsm
+    HDDSIZEGB: 40
+    YUI_REST_API: 1
+schedule:
+    - installation/bootloader_start
+    - installation/setup_libyui
+    - installation/access_beta_distribution
+    - installation/product_selection/install_SLES
+    - installation/licensing/accept_license
+    - installation/registration/register_via_scc
+    - installation/module_selection/select_module_transactional
+    - installation/add_on_product/skip_install_addons
+    - installation/system_role/accept_selected_role_transactional_server
+    - installation/partitioning/accept_proposed_layout
+    - installation/clock_and_timezone/accept_timezone_configuration
+    - installation/authentication/use_same_password_for_root
+    - installation/authentication/default_user_simple_pwd
+    - installation/resolve_dependency_issues
+    - installation/installation_settings/validate_ssh_service_enabled
+    - installation/installation_settings/open_ssh_port
+    - installation/installation_settings/validate_default_target
```

Comment:
here there is not validation: https://openqa.suse.de/tests/7716347#step/installation_overview/13